conversion/conversion.go

```diff
@@ -417,6 +417,70 @@ func getSeekable(f *os.File) (*os.File, int64, error) {
 	return fcopy, n, nil
 }
 
+func VerifyTargetDb(targetDb, project, instance, dbName string) (string, bool, error) {
+	ctx := context.Background()
+	adminClient, err := database.NewDatabaseAdminClient(ctx)
+	if err != nil {
+		return "", false, fmt.Errorf("can't create admin client: %w", analyzeError(err, project, instance))
+	}
+	defer adminClient.Close()
+
+	dbPath := fmt.Sprintf("projects/%s/instances/%s/databases/%s", project, instance, dbName)
+	_, err = adminClient.GetDatabase(ctx, &adminpb.GetDatabaseRequest{Name: dbPath})
+	if err != nil {
+		// Differentiate between "db does not exist" and other errors? Assuming err means db does not exist for now
+		return targetDb, true, nil
+	}
+	//if resp.DatabaseDialect == "POSTGRESQL" {
+	//	targetDb = TARGET_EXPERIMENTAL_POSTGRES
+	//} else {
+	//	targetDb = TARGET_SPANNER
+	//}
+	dbDdl, err := adminClient.GetDatabaseDdl(ctx, &adminpb.GetDatabaseDdlRequest{Database: dbPath})
+	if err != nil {
+		fmt.Printf("\nCan't fetch database ddl: %v\n", err)
+		return "", false, fmt.Errorf("can't fetch database ddl")
+	}
+	if len(dbDdl.Statements) != 0 {
+		fmt.Printf("\nWriting to existing databases with non-empty ddl not supported\n")
+		return "", false, fmt.Errorf("Writing to existing databases with non-empty ddl not supported")
+	}
```

Comment:
Note: When running is schema-only mode with a dbname that already exists with some DDL, we will end up throwing this error.

Reply:
I think it's fair to call out when customers point to an existing Spanner database even in schema-only mode. I am though surprised that we allow users to pass in targetDb name in schema_only mode?

Reply:
Yes se do allow passing of the DBname in schema-only currently, I personally pass it with schema-only mode for more readable file names compared to pgdump-<date>-<hash>